Week 1/ES6.js

```diff
@@ -1,4 +1,4 @@
 $("button").on("click", () => {
 	var photo = $('#picture_URL').val();
-  	$('#photo').append('<img src=' + photo + '>')
+  	$('#photo').append('<img src='+ photo +'>')
```

Comment:
Looks good, but let's use some ES6. Use `const` instead of `var` since we won't be changing the value of photo again.

```js
$("#photo").append(`<img src='${photo}'/>`);
```